fe/fe-core/src/main/java/com/starrocks/journal/JournalEntity.java

```diff
@@ -184,6 +185,11 @@ public void readFields(DataInput in) throws IOException {
                 isRead = true;
                 break;
             }
+            case OperationType.OP_ERASE_MULTI_TABLES: {
```

Comment:
Better to remove the OP_ERASE_TABLE writing function. And comment OP_ERASE_TABLE to deprecated.